local-cli/util/yarn.js

```diff
@@ -8,6 +8,7 @@
  */
 'use strict';
 
+const execSync = require('child_process').execSync;
```

Comment:
is it used in `line 25 and 27`.

https://github.com/facebook/react-native/pull/11174/files/48039bd0dd7404d53e27af458bd73691f81048a9#diff-549f6559732f42d8d33684b24c5cea46R25